src/edit.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-cycle */
+// import updateStorage from './storage';
```

Comment:
- [ ] [OPTIONAL] Remember that you don't need to commit commented code, please remove it